docker/wsgi.py

```diff
@@ -50,20 +61,21 @@ def read_models_from_file():
                 if line.startswith('['):
                     found_mms_args = 1 if mms_arg_header.lower() in line.lower() else 0
                 if found_mms_args is 1:
-                    if line.startswith('--') and content[i+1] != 'optional':
-                        args.append(line)
-                        if line.startswith('--model'):
-                            args += content[i + 1].split(' ')
-                        else:
-                            args.append(content[i + 1])
+                    if line.startswith('--num-GPUs') and content[i+1] != 'optional':
+                        num_gpu = int(content[i+1])
```

Comment:
So if the num-gpus is given with a non-optional value, we wouldn't be reading models? They seem to be in a if/else statement. Also, default the "num_gpu" variable to "1" or some value here and use that in the "get_gpu_ids" def. Don't assign "num_gpu" inside the "get_gpu_ids" method.

Reply:
The models need to be read only in case where the line starts with --model. Here the  line starts with -- num-gpus, so we don't need to read model for that line